CHANGE_LOG

```diff
@@ -1,7 +1,18 @@
 Version 0.3
 -----------
+This release adds:
 
-In development
+* Dynamic generation of `scipy.special.signatures` (with thanks to Brandon T.
+  Willard)
+* Github Actions CI
+
+Importantly, the dynamic generation of the signatures makes numba-scipy now
+compatible with multiple SciPy releases simultaneously.
+
+Contributors
+* Brandon T. Willard
+* Stuart Archibald
```

Comment:
I don't recall adding a patch?!

Reply:
Your name appears in the git-log:

```
 💣 zsh» git --no-pager shortlog -sn 0.3.0dev0..
    24	Valentin Haenel
     7	esc
     4	Brandon T. Willard
     2	stuartarchibald
```

Probably a merge?

Reply:
Yes it was merges:

```
 💣 zsh» git --no-pager shortlog -sn --no-merges 0.3.0dev0..
    24	Valentin Haenel
     4	Brandon T. Willard
     1	esc
```